README.md

```diff
@@ -1,27 +1,27 @@
 <!-- Update the title -->
-# Terraform Modules Template Project
+# WebSphere Liberty operator on Red Hat OpenShift VPC cluster module
 
 <!--
 Update status and "latest release" badges:
   1. For the status options, see https://terraform-ibm-modules.github.io/documentation/#/badge-status
   2. Update the "latest release" badge to point to the correct module's repo. Replace "terraform-ibm-module-template" in two places.
 -->
-[![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
+[![Implemented (No quality checks)](https://img.shields.io/badge/Status-Implemented%20(No%20quality%20checks)-yellowgreen)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
 [![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-websphere-liberty-operator?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-websphere-liberty-operator/releases/latest)
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 <!-- Add a description of module(s) in this repo -->
-TODO: Replace me with description of the module(s) in this repo
+Use this module to install a WebSphere® Liberty operator and create an instance of WebSphere Liberty operator on IBM Cloud Red Hat OpenShift cluster on VPC Gen2.
```

Comment:
we'll need to check the trademarks and naming guidelines. I'm not sure how Liberty fits into 

For Websphere we probably want to use 
- **IBM&reg; WebSphere Application Server for IBM Cloud&reg;** on first use
- and **WebSphere Application Server** on subsequent uses.

do we still say Gen2? Or are we installing this on the landing zone? If SLZ, then use the name *Red Hat OpenShift Container Platform on VPC landing zone*.

Reply:
I go this answer about names: 

> The official name for the WAS Liberty product is "WebSphere Application Server Liberty" but the operator content has always called the operator "WebSphere Liberty operator" or "IBM WebSphere Liberty operator" (no WAS).